Dockerfile

```diff
@@ -38,5 +39,9 @@ RUN mkdir ${KAFKA_HOME} && apt-get update && apt-get install curl -y && apt-get
 COPY --from=kafka_dist /var/tmp/kafka_$scala_version-$kafka_version ${KAFKA_HOME}
 
 RUN chmod a+x ${KAFKA_HOME}/bin/*.sh
+RUN chmod g+rwX ${KAFKA_HOME}
+
+# RUN adduser ${PROJECT_NAME} --disabled-password
+USER 1337
```

Comment:
This won't work for istio-proxy sidecar injected Kafka brokers

1337 is the UID that istio-proxy runs as and can't have anything else running with that UID due to iptables rules.

https://istio.io/latest/docs/reference/config/analysis/ist0144/

Reply:
Is there any specific ID you recommend?

Reply:
note in k8s RHOS we'll need to set "runAsUser: 10001` (or whatever matching UID) and we can use `nonroot-v2` policy